deepchecks/vision/README.rst

```diff
@@ -144,8 +141,7 @@ Example for a model evaluation check calculating mAP:
       <h4>Mean Average Precision Report</h4>
       <p>Summarize mean average precision metrics on a dataset
       and model per IoU and area range.</p>
-      <a href="https://https://docs.deepchecks.com/en/stable/examples/vision/checks/performance/examples/plot_mean_average_precision_report.html
-      utm_source=display_output&utm_medium=referral&utm_campaign=readme_check_link" target="_blank">
+      <a href="https://docs.deepchecks.com/en/stable/checks_gallery/vision/model_evaluation/plot_mean_average_precision_report.html#sphx-glr-checks-gallery-vision-model-evaluation-plot-mean-average-precision-report-py" target="_blank">

```

Comment:
```suggestion
      <a href="https://docs.deepchecks.com/en/stable/checks_gallery/vision/model_evaluation/plot_mean_average_precision_report.html" target="_blank">

```

Reply:
better to link to the top of the page (so that also the link for downloading the code is clearer)

Reply:
> better to link to the top of the page (so that also the link for downloading the code is clearer)

Should I have to make the mentinoned change ?

Reply:
Yes you can just commit my suggestions before merging

Reply:
Sure